.github/workflows/windows.yml

```diff
@@ -22,11 +22,11 @@ jobs:
           Install-WindowsFeature -name Hyper-V-PowerShell
 
           $env:CSI_PROXY_GH_ACTIONS="TRUE"
-          go test --timeout 20m -v -race ./integrationtests/...
```

Comment:
Is there a reason `-race` was removed?

Reply:
Re-enabled `-race` but with `checkptr=0` disabled as part of https://github.com/kubernetes-csi/csi-proxy/pull/385
This should be related to dereferencing the pointer allocated by COM is not considered as safe in Go.
We either need to use C/CGO on these pointers or set `checkptr=0`.